cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup.adoc

```diff
@@ -0,0 +1,227 @@
+----
+. Run `rosa version` to verify successful installation.
+
+== Installing the OpenShift CLI
+There are a few ways to install the OpenShift CLI (`oc`):
+
+* *Option 1- Using the ROSA CLI:*
+.. Run `rosa download oc`.
+.. Once downloaded, unzip the file and move the executables into a directory in your path.
+* *Option 2- Using the Openshift documentation:*
+.. Follow the directions on the xref:../../cli_reference/openshift_cli/getting-started-cli.adoc#installing-openshift-cli[documentation page]
+* *Option 3- Using your OpenShift cluster:*
+.. If you already have an OpenShift cluster, you can access the CLI tools page by clicking on the *Question mark*, then *Command Line Tools*.
++
+image::cloud_experts_getting_started_setup_cli_tools.png[]
+
+.. Then, download the relevant tool for your operating system.
+
+=== Using `oc` instead of `kubectl`
+While `kubectl` can be used with an OpenShift cluster, `oc` is specific to OpenShift. It includes the standard set of features from `kubectl` as well as additional support for OpenShift functionality. For more information, see xref:../../cli_reference/openshift_cli/usage-oc-kubectl.adoc#usage-oc-kubectl[Usage of oc and kubectl commands].
+
+== Configuring the AWS CLI
+To configure the AWS CLI, follow these steps:
+
+. Enter `aws configure` in the terminal.
+. Enter your AWS access key ID and press enter.
+. Enter your AWS secret access key and press enter.
+. Enter the default region in which you want to deploy.
+. Enter the desired output format, specifying either `table` or `json`.
+
+.Example output
+[source, terminal]
+----
+$ aws configure
+AWS Access Key ID: AKIA0000000000000000
+AWS Secret Access Key: NGvmP0000000000000000000000000
+Default region name: us-east-1
+Default output format: table
+----
+
+== Verifying the configuration
+Verify that the configuration is correct by following these steps:
+
+. Run the following command to query the AWS API:
++
+[source,terminal]
+----      
+aws sts get-caller-identity
+----
+. You should see a table or JSON file. Verify that the account information is correct.
++
+.Example output
++
+[source, terminal]
+----
+$ aws sts get-caller-identity
+------------------------------------------------------------------------------
+|                                GetCallerIdentity                           |
++--------------+----------------------------------------+--------------------+
+|    Account   |                   Arn                  |        UserId      |
++--------------+----------------------------------------+--------------------+
+|  000000000000|  arn:aws:iam::00000000000:user/myuser  |  AIDA00000000000000|
++--------------+----------------------------------------+--------------------+
+----
+
+== Ensuring the ELB service role exists
+[TIP]
+====
+Make sure that the service role for the ELB already exists, otherwise cluster deployment could fail.
+====
+
+Run the following command to check for the ELB service role and create it if it is missing:
```

Comment:
I think these steps should be numbered, similar to the "Verifying the configuration" section.